src/preload/script.ts

```diff
@@ -1,15 +1,15 @@
-import { addForceDisabled } from "../panic/panic";
-import moduleReplacements from "./moduleReplacements";
-import { insertElement, replaceElement } from "./replaceElement";
-import { fullReplacementCached } from "./replacementHelpers/cacheReplacement";
-import window from "#globals";
+import window, { Console } from "#globals";
 import injectScript from "#utils/injectScript.ts";
 import {
   postMessageUp,
   listenToMessageDown,
   arrayToSet,
 } from "#utils/messages.ts";
 import { pollForValue } from "#utils/utils.ts";
+import { addForceDisabled } from "../panic/panic";
+import moduleReplacements from "./moduleReplacements";
+import { insertElement, replaceElement } from "./replaceElement";
+import { fullReplacementCached } from "./replacementHelpers/cacheReplacement";
```

Comment:
Looks like these imports got auto-sorted? I took out prettier-plugin-sort-imports because it failed with some update and/or caused confusion due to the `#` introduction (we might want to add it back at some time, but I haven't seen it super helpful). An `npm clean-install` might remove it.

Reply:
I was a bit confused about this too. My local prettier did that and it passed linter, so I just left it.